esmvaltool/cmorizers/obs/cmorize_obs_cds_satellite_lai_fapar.py

```diff
@@ -0,0 +1,217 @@
+"""ESMValTool CMORizer for cds-satellite-lai-fapar data.
+
+Tier
+   Tier 3
+Source
+   https://cds.climate.copernicus.eu/cdsapp#!/dataset/satellite-lai-fapar?tab=form
+Last access
+   20190703
+
+Download and processing instructions
+   - Open in a browser the data source as specified above
+   - Put the right ticks:
+      - Tick variables FAPAR and LAI
+      - Tick SPOT
+      - Tick V1
+      - Tick all available years
+      - Tick all available months
+      - Tick Nominal day 20
+   - Click 'submit form'
+   - According to ESMValTool practice, put them in the right rawobsdir folder
```

Comment:
When trying this, I get an error that the request is too large and should be split.
Maybe add a caveat on this, suggesting e.g. to download each year separately.